DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommandLineTool/AnonymizerOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using CommandLine;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Tool
+{
+    public class AnonymizerOptions
+    {
+        [Option('i', "inputFile", Required = false, HelpText = "Input DICOM file")]
+        public string InputFile { get; set; }
+
+        [Option('o', "outputFile", Required = false, HelpText = "Output DICOM file")]
+        public string OutputFile { get; set; }
+
+        [Option('c', "configFile", Required = false, Default = "configuration.json", HelpText = "Anonymization configuration file path.")]
+        public string ConfigurationFilePath { get; set; }
+
+        [Option('I', "inputFolder", Required = false, HelpText = "Input folder")]
+        public string InputFolder { get; set; }
+
+        [Option('O', "outputFolder", Required = false, HelpText = "Output folder")]
+        public string OutputFolder { get; set; }
+
+        [Option('v', "autoValidate", Required = false, Default = true, HelpText = "Auto validate output value when anonymizing.")]
+        public bool AutoValidate { get; set; }
```

Comment:
This name is a bit confusing. What do we mean by `auto`? And are we only going to validate only the output if this option is set to true?

Reply:
I have change this option to validateOuput.
(since we validate the ouput after each operation before, now we change this behavior and validate output after all operations)